lib/Slider/Slider.js

```diff
@@ -579,6 +586,10 @@ module.exports = kind(
 			this.updatePopup(this.elasticFrom);
 			this.set('value',this.elasticFrom);
 
+			if (this.enableJumpIncrement) {
+				this.updateButtonStatus(v);
```

Comment:
I also thought of performance issue about updating status while dragging. But I didn't know which is better way. If updating the status when drag is finished, we can get better performance but the status is not changed while dragging. I think we can go with your suggestion because Robert doesn't have objection about this. Right?